components/navbar/NavLink.js

```diff
@@ -9,7 +9,7 @@ export default function NavLink({ path, item, mode, setIsOpen, onClick }) {
       "text-gray-300 hover:ring-2 hover:ring-gray-700 hover:text-white block px-3 py-2 rounded-md text-base font-medium";
   }
 
-  if (path === item.url) {
+  if (path?.split("/")[1] === item.url.replace("/", "")) {
```

Comment:
I think it would be better to do a similar match on each side using the split and the 2nd item (`[1]`)?

Reply:
I think you are right, at some point we need to check the other side also. But currently, we are just focusing on making an active state to the navbar menu list.

Reply:
I think this is a straight forward improvement now, which would reduce having a bug in the future

Reply:
> I think this is a straight forward improvement now, which would reduce having a bug in the future

I have updated the code based on matching each side.